ktor-io/posix/src/io/ktor/utils/io/ByteReadChannelNative.kt

```diff
@@ -75,12 +75,14 @@ public actual interface ByteReadChannel {
      * Reads all available bytes to [dst] buffer and returns immediately or suspends if no bytes available
      * @return number of bytes were read or `-1` if the channel has been closed
      */
+    @OptIn(ExperimentalForeignApi::class)
     public suspend fun readAvailable(dst: CPointer<ByteVar>, offset: Int, length: Int): Int
 
     /**
      * Reads all available bytes to [dst] buffer and returns immediately or suspends if no bytes available
      * @return number of bytes were read or `-1` if the channel has been closed
      */
+    @OptIn(ExperimentalForeignApi::class)
     public suspend fun readAvailable(dst: CPointer<ByteVar>, offset: Long, length: Long): Int
```

Comment:
is it okay to have an OptIn annotation in a patch release?

Reply:
yep, it's kotlin native and triggered by the kotlin version